CONTRIBUTING.md

```diff
@@ -5,9 +5,8 @@ There are several github actions in this repo to do some basic checks with pull
 | Action Name                                                  | Note                         |
 | ------------------------------------------------------------ | ---------------------------- |
 | [codeql-analysis](https://github.com/Azure-Samples/azure-spring-boot-samples/blob/main/.github/workflows/codeql-analysis.yml) | Code analysis                |
-| [check-style](https://github.com/Azure-Samples/azure-spring-boot-samples/tree/main/.github/workflows) | Java check style             |
-| [check-markdown-links](https://github.com/Azure-Samples/azure-spring-boot-samples/blob/main/.github/workflows/check-markdown-links.yml) | Check broken markdown links. |
-| [azure-spring-boot-samples-action](https://github.com/Azure-Samples/azure-spring-boot-samples/blob/main/.github/workflows/azure-spring-boot-samples-action.yml) | Maven build check.           |
+| [check-markdown-links](https://github.com/Azure-Samples/azure-spring-boot-samples/blob/main/.github/workflows/markdown-link-check.yml) | Check broken markdown links. |
```

Comment:
Do we need to list the actions here? How about considering listing the badges, or simply adding the badges to the main/README.md?

![CodeQL](https://github.com/Azure-Samples/azure-spring-boot-samples/actions/workflows/codeql-analysis.yml/badge.svg)

Reply:
I will drop this pr, it will be updated in other prs, the pr here is only used for testing.